Examples/Swift/ViewController.swift

```diff
@@ -242,6 +242,34 @@ class ViewController: UIViewController, MGLMapViewDelegate {
         present(navigationViewController, animated: true, completion: nil)
     }
     
+    func navigationMapView(_ mapView: MGLMapView, viewFor annotation: MGLAnnotation) -> MGLAnnotationView? {
+        guard annotation is MGLUserLocation else { return nil }
```

Comment:
The reason is that `MGLMapView.showsUserLocation` is set during storyboard initialization, [causing MGLMapView to call `MGLMapViewDelegate.mapView(_:viewFor:)`](https://github.com/mapbox/mapbox-gl-native/blob/3a3a072b7a34e949b9bcc0188c4048e61cdf24dc/platform/ios/src/MGLMapView.mm#L4184). Unfortunately, `navigationMapView` isn’t set until much later.